pkg/trafficpolicy/utils.go

```diff
@@ -0,0 +1,17 @@
+package trafficpolicy
+
+// subset returns true if the second array is completely contained in the first array
+func subset(first, second []string) bool {
```

Comment:
maybe just drop this function in the same file where it is used or in the generic utility package?

Reply:
For now I moved it into the file where its being used